Cargo.toml

```
@@ -18,10 +18,10 @@ description = "A pure-Rust implementation of Bulletproofs using Ristretto"
 edition = "2018"
 
 [dependencies]
-curve25519-dalek = { version = "4.0.0-pre.1", default-features = false, features = ["u64_backend", "serde"] }
+curve25519-dalek = { version = "4.0.0-pre.2", default-features = false, features = ["u64_backend", "serde"] }
 subtle = { version = "2", default-features = false }
-sha3 = { version = "0.9.1", default-features = false }
-digest = { version = "0.9.0", default-features = false }
+sha3 = { version = "0.10", default-features = false }
+digest = { version = "0.10", default-features = false }
```

Digest only has `core-api` as [a default-feature 0.10](https://docs.rs/crate/digest/latest/features), so this can be simplified:
```suggestion
digest = "0.10"
```